speech/snippets/transcribe_gcs_v2_test.py

```diff
@@ -14,37 +14,23 @@
 
 import os
 import re
-from uuid import uuid4
 
 from google.api_core.retry import Retry
-from google.cloud.speech_v2 import SpeechClient
-from google.cloud.speech_v2.types import cloud_speech
 import pytest
 
 import transcribe_gcs_v2
 
 
-def delete_recognizer(name: str) -> None:
-    client = SpeechClient()
-    request = cloud_speech.DeleteRecognizerRequest(name=name)
-    client.delete_recognizer(request=request)
-
-
 @Retry()
 def test_transcribe_gcs_v2(capsys: pytest.CaptureFixture) -> None:
     project_id = os.getenv("GOOGLE_CLOUD_PROJECT")
 
-    recognizer_id = "recognizer-" + str(uuid4())
     response = transcribe_gcs_v2.transcribe_gcs_v2(
-        project_id, recognizer_id, "gs://cloud-samples-data/speech/audio.flac"
+        project_id, "gs://cloud-samples-data/speech/audio.flac"
```

Comment:
Consider making this a module_level variable: _TEST_AUDIO_FILE_PATH = "gs://..."